pages.bn/android/input.md

```diff
@@ -0,0 +1,25 @@
+# input
+
+> একটি Android ডিভাইসে ইভেন্ট কোড বা টাচস্ক্রিন অঙ্গভঙ্গি পাঠান।
+> এই কমান্ডটি শুধুমাত্র `adb shell` এর মাধ্যমে ব্যবহার করা যেতে পারে।
+> আরও তথ্য পাবেন: <https://developer.android.com/reference/android/view/KeyEvent.html#constants_1>.
+
+- একটি Android ডিভাইসে একটি একক অক্ষরের জন্য একটি ইভেন্ট কোড পাঠান:
+
+`input keyevent {{event code}}`
+
+- একটি অ্যান্ড্রয়েড ডিভাইসে একটি পাঠ্য পাঠান (`%s` স্পেস প্রতিনিধিত্ব করে):
+
+`input text "{{লেখা}}"`
+
+- একটি Android ডিভাইসে একটি একক ট্যাপ পাঠান:
+
+`input tap {{x_pos}} {{y_pos}}`
+
+- একটি Android ডিভাইসে একটি সোয়াইপ অঙ্গভঙ্গি পাঠান:
+
+`input swipe {{x_start}} {{y_start}} {{x_end}} {{y_end}} {{duration_in_ms}}`
```

Comment:
```suggestion
- একটি Android ডিভাইসে একটি সোয়াইপ অঙ্গভঙ্গি পাঠান:

`input swipe {{x_শুরু}} {{y_শুরু}} {{x_শেষ}} {{y_শেষ}} {{1..অনন্ত}}`
```

Almost the same issue as above.
Optional fix (error source: original page): unexpected generic placeholder, range syntax expected.
Warning: no valid number range specified, `1..infinity` used as a fallback, documentation issue.